src/properties/practicebot/behavior.properties

```diff
@@ -12,7 +12,7 @@ gearintake.speed.out = .3
 #Shooter Subsystem
 agitator.speed.in = -.5
 agitator.speed.out = .6
-feeder.speed.in = .5
+feeder.speed.in = -.5
```

Comment:
Instead of having a negatives here, we could negate the motor direction at subsystem or talon firmware level. It doesn't matter which way you do it, but keeping it consistent is important.